src/progress-bar.tsx

```diff
@@ -50,7 +50,7 @@ export const ProgressBar = ({
                   'aria-valuenow': progressValue,
```

Comment:
The ticket description suggested removing `aria-valuenow`, but removing it makes the progressbar indeterminate for screen readers, so it must be there.

When `aria-valuenow` is set, the screen reader reads it after the aria-label, resulting in a repeated read because the `aria-label` already includes the value. The best way to avoid this is using `aria-valuetext` instead of `aria-label`